.travis.yml

```diff
@@ -0,0 +1,122 @@
+    #       the discussion on #121).  As an example, the set chosen below covers
+    #       the most used distros families, systemd and non-systemd and the latest
+    #       three supported Saltstack versions with python2 and 3.
+    #       As for `kitchen.yml`, that should still contain all of the platforms,
+    #       to allow for comprehensive local testing
+    #       Ref: https://github.com/saltstack-formulas/template-formula/issues/118
+    #       Ref: https://github.com/saltstack-formulas/template-formula/issues/121
+    - env: INSTANCE=default-debian-10-master-py3
+    # - env: INSTANCE=default-ubuntu-1804-master-py3
+    # - env: INSTANCE=default-centos-8-master-py3
+    # - env: INSTANCE=default-fedora-31-master-py3
+    # - env: INSTANCE=default-opensuse-leap-151-master-py3
+    # - env: INSTANCE=default-amazonlinux-2-master-py2
+    # - env: INSTANCE=default-arch-base-latest-master-py2
+    # - env: INSTANCE=default-debian-10-2019-2-py3
+    # - env: INSTANCE=default-debian-9-2019-2-py3
+    - env: INSTANCE=default-ubuntu-1804-2019-2-py3
+    # - env: INSTANCE=default-centos-8-2019-2-py3
+    # - env: INSTANCE=default-fedora-31-2019-2-py3
+    # - env: INSTANCE=default-opensuse-leap-151-2019-2-py3
+    # - env: INSTANCE=default-centos-7-2019-2-py2
+    - env: INSTANCE=default-amazonlinux-2-2019-2-py2
+    - env: INSTANCE=default-arch-base-latest-2019-2-py2
+    - env: INSTANCE=default-fedora-30-2018-3-py3
+    # - env: INSTANCE=default-debian-9-2018-3-py2
+    # - env: INSTANCE=default-ubuntu-1604-2018-3-py2
+    # - env: INSTANCE=default-centos-7-2018-3-py2
+    - env: INSTANCE=default-opensuse-leap-151-2018-3-py2
+    # - env: INSTANCE=default-amazonlinux-2-2018-3-py2
+    # - env: INSTANCE=default-arch-base-latest-2018-3-py2
+    # - env: INSTANCE=default-debian-8-2017-7-py2
+    # - env: INSTANCE=default-ubuntu-1604-2017-7-py2
+    - env: INSTANCE=centos6-centos-6-2017-7-py2
+    # - env: INSTANCE=default-fedora-30-2017-7-py2
+    # - env: INSTANCE=default-opensuse-leap-151-2017-7-py2
+    # - env: INSTANCE=default-amazonlinux-2-2017-7-py2
+    # - env: INSTANCE=default-arch-base-latest-2017-7-py2
+
+    ## Define the release stage that runs `semantic-release`
+    - stage: 'release'
+      language: 'node_js'
+      node_js: 'lts/*'
+      env: 'Release'
+      name: 'Run semantic-release inc. file updates to AUTHORS, CHANGELOG & FORMULA'
+      before_install: 'skip'
+      script:
+        # Update `AUTHORS.md`
+        - export MAINTAINER_TOKEN=${GH_TOKEN}
+        - go get github.com/myii/maintainer
+        - maintainer contributor
+
+        # Install all dependencies required for `semantic-release`
+        - npm i -D @semantic-release/changelog@3
+                   @semantic-release/exec@3
+                   @semantic-release/git@7
+      deploy:
+        provider: 'script'
+        # Using deprecated `skip_cleanup` until `cleanup: false` works reliably
+        # cleanup: false
+        skip_cleanup: true
```

Comment:
With the merge of https://github.com/saltstack-formulas/template-formula/pull/178, these changes can now be applied here:

* https://github.com/saltstack-formulas/template-formula/pull/178/files#diff-354f30a63fb0907d4ad57269548329e3L118-R121

```diff
-        # Using deprecated `skip_cleanup` until `cleanup: false` works reliably
-        # cleanup: false
-        skip_cleanup: true
+        # Opt-in to `dpl v2` to complete the Travis build config validation (beta)
+        # * https://docs.travis-ci.com/user/build-config-validation
+        # Deprecated `skip_cleanup` can now be avoided, `cleanup: false` is by default
+        edge: true
```